packages/docs/docs/guides/prompting.md

```diff
@@ -41,3 +41,50 @@ Output:
 ```
 
 The problem is that the model is predicting that a question about one state is often followed by questions about other states.
+
+## Primitives to get you prompting faster
+
+We have included a small set of prompts that we found useful in (`packages/ai-jsx/src/core/prompts.tsx`).
+You can use them either as shortcuts, or as a starting point if you are new to prompting:
+
+```tsx
+function App() {
+  // Asking the model to respond as if they were a React developer
+  return (
+    <ChatCompletion>
+      <Prompt persona="a React developer" />
```

Comment:
If the idea is that the `Prompt` is a drop-in `SystemMessage`, then I'd name it something different – maybe `PersonaSystemMessage`?

When I first had the idea of naming something `Prompt`, I thought it would look more like:

```tsx
<SystemMessage>
  <Prompt hhh />
  Answer using this context: <DocsQA />
</SystemMessage>
```

Reply:
You can still do the above too, right?
From what I understood, everything except for the parent Message type gets resolved to the children, so you can just as easily write:
```tsx
<UserMessage>
  <Prompt hhh />
</UserMessage>
```
And it would become a `UserMessage`. It feels to me like I'm just creating a default: if they don't specify, it's going to be a `SystemMessage`.

Reply:
Yep, I agree that would work – it just feels weird to see it `Prompt` written as a top-level child of `ChatCompletion`. But it's not that big a deal. No need to change it until we see if we get more feedback to this extent.